next/src/components/sections/DocumentsSection/DocumentsSection.tsx

```diff
@@ -0,0 +1,107 @@
+import { useTranslation } from 'next-i18next'
+import React, { Fragment } from 'react'
+
+import DocumentRowCard from '@/src/components/cards/DocumentRowCard'
+import HorizontalDivider from '@/src/components/common/Divider/HorizontalDivider'
+import SectionContainer from '@/src/components/layouts/SectionContainer'
+import SectionHeader from '@/src/components/layouts/SectionHeader'
+import DocumentsAll from '@/src/components/sections/DocumentsSection/DocumentsAll'
+import { DocumentsSectionFragment } from '@/src/services/graphql'
+import { formatDate } from '@/src/utils/formatDate'
+import { formatFileExtension } from '@/src/utils/formatFileExtension'
+import { formatFileSize } from '@/src/utils/formatFileSize'
+import { isDefined } from '@/src/utils/isDefined'
+import { useLocale } from '@/src/utils/useLocale'
+
+type Props = {
+  section: DocumentsSectionFragment
+}
+
+/**
+ * Figma OLO: https://www.figma.com/design/2qF09hDT9QNcpdztVMNAY4/OLO-Web?node-id=1932-18019&t=U7rn1Il95Xd9GkCS-0
+ */
+
+const DocumentsSection = ({ section }: Props) => {
+  const { t } = useTranslation()
+  const locale = useLocale()
+
+  const { title, text, documents, showAll } = section
+
+  const filteredDocuments = documents?.data.filter(isDefined) ?? []
+
+  return (
+    <SectionContainer>
+      {showAll ? (
+        <DocumentsAll />
+      ) : (
```

Comment:
I just want to point out that it was more consistent with OLO before this change 🤷